app/config/parameters.yml.dist

```diff
@@ -12,6 +12,8 @@ parameters:
     #
     # database_url: mysql://root:pass@127.0.0.1:3306/symfony_demo
     #
+    # Furthermore, you must remove or comment out lines 11-14 from config_dev.yml regarding SQLite!
```

Comment:
I think it's better to write: `Furthermore, you must remove or comment out the "doctrine" section from config_dev.yml regarding SQLite!`